src/content/docs/en/tutorial/5-astro-api/2.mdx

```diff
@@ -174,12 +174,13 @@ Revisit each of your existing Markdown pages and ensure that every post contains
 
 Add the following code to provide you with a list of every tag used in your blog posts.
 
-  ```astro title="src/pages/tags/[tag].astro" ins={7}
+  ```astro title="src/pages/tags/[tag].astro" ins={8}
```

Comment:
El ejemplo creo que no está mal ya que más abajo de la página se muestra la parte de codigo faltante y aún mas abajo se muestra el codigo entero https://docs.astro.build/en/tutorial/5-astro-api/2/#2-create-an-array-of-all-your-existing-tags

Reply:
Es cierto, lo había interpretado mal. Gracias por la aclaración 👍